### ISSUE TITLE
EPIC: Creating issues to replace "messages" with "archives" in project Slack links

### ISSUE THREAD

**Issue by jdingeman** [hackforla/website#4564]
### Dependency
- [x] #5716
- [x] #6511
- [x] #6512
- [x] #6513
- [x] #6514
- [x] #6515

### Overview
We need to edit the Slack links for each [project](https://github.com/hackforla/website/tree/gh-pages/_projects) so that we have continuity between project Slack links. 

### Action Items
- [x] In the Dependency section of this issue, click on each item to create the issue with the correct name (one at a time)
  - [x] The issue "Project Slack links: Civic Tech Jobs" has already been written, which you can reference as an example.
- [x] Paste the template below (in the Template section) into the issue you just created. 
  - [x] Then, edit the "Action Items" and "Resources" sections as follows:
    - [x] [Replace with PROJECT NAME] will be replaced with the corresponding project name (e.g. Civic Tech Jobs)
    - [x] [Replace PROJECT.MD file name] will be replaced with the corresponding project's Markdown file name (e.g. civic-tech-jobs.md )
    - [x] [Replace with EXISTING SLACK LINK] will be replaced with the existing Slack link for the project (e.g. https://hackforla.slack.com/messages/C02509WHFQQ). The project's Slack link is the Slack `url` under the `links` section. (e.g. https://github.com/hackforla/website/blob/3a5c1479d7a51730fa39a2aeab2e5c2435f7b42d/_projects/civic-tech-jobs.md?plain=1#L99-L105)
    - [x] [Replace with NEW SLACK LINK] will be replaced with the new Slack link for the project, where
       - [x]  `messages` in the existing link is replaced with `archives` (e.g. https://hackforla.slack.com/archives/C02509WHFQQ)
    - [x] [Replace PROJECT PAGE URL] will be replaced with the corresponding project page URL (e.g. civic-tech-jobs in https://www.hackforla.org/projects/civic-tech-jobs)
    - [x] [Replace with LINK TO THE PROJECT PAGE IN THE GITHUB REPO] will be replaced with the link to the Markdown file in the "_projects" directory (e.g. https://github.com/hackforla/website/blob/gh-pages/_projects/civic-tech-jobs.md?plain=1)
  - [x] Add the labels `role: front end`, `role: back end/devOps`, `good first issue`, `size: 0.25pt`, `P-Feature: Project Info and Page`, `P-Feature: Projects page`, `ready for dev lead`
  - [x] Remove the labels `Complexity: Missing`, `Feature: Missing`, `role missing`, `size: missing`
  - [x] Add the issue to the Project Board
  - [x] Add the milestone `03.02 Onboarding flow`
- [x] Once all the issues have been created, in this issue, 
  - [x] Change the heading `### Issues to Make` to `### Dependency`
  - [x] Add a `dependency` label
  - [x] In a comment to @hackforla/website-merge, ask a Merge Team member to review the issues you created to see if they are ready for prioritization. Then, add a `ready for dev lead` label and move this issue to the Questions / In Review column.
  - [x] Once you've received Merge Team member approval on the issues you wrote, move this issue to the Ice Box column
- [x] If all the dependency issues are closed, remove the dependency label, move this issue from the Ice Box to the In Progress column, and close this issue

#### Template
Copy this template:
````
### Prerequisite
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started page](https://www.hackforla.org/getting-started) and attend an onboarding session.
2. Before you claim or start working on an issue, please make sure you have read our [How to Contribute to Hack for LA Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md).

### Overview
We need to edit the Slack link for the project so that we have continuity between project Slack links.

### Action Items
- [ ] In `_projects/[Replace PROJECT.MD file name]`, for the Slack `url` under the `links` section, change 

From:
```
url: [Replace with EXISTING SLACK LINK]
```
To:
```
url: [Replace with NEW SLACK LINK]
```
- [ ] Ensure that clicking the Slack link resolves to the new address at the following places:
  - [ ] On the [Projects page](https://www.hackforla.org/projects/), the Slack link under the `Links` field for the project's card
  - [ ] On the [Replace with PROJECT NAME] page (https://www.hackforla.org/projects/[Replace PROJECT PAGE URL]),
    - [ ] the Slack link under the `Links` field in the Project Overview section
    - [ ] the Slack link in the Resources/Instructions section 
- [ ] Once your pull request is merged, go to #4564 and
  - [ ] Check off this issue under the _Dependency_ section
  - [ ] If all the issues in the _Dependency_ section are checked off, move #4564 to **Questions / In Review** column and add a **ready for dev lead** label to it

### Resources/Instructions
- Directory to find the page once you have it in your IDE: `_projects/[Replace PROJECT.MD file name]`
- [Projects page](https://www.hackforla.org/projects/)
- Project page URL: https://www.hackforla.org/projects/[Replace PROJECT PAGE URL]
- Click on the filename below to see the source file code:
[Replace with LINK TO THE PROJECT PAGE IN THE GITHUB REPO]
- This issue is tracked in #4564
````

### Resources/Instructions

- Associated ER for this issue: #4535

<details><summary>Finding the Slack Channel name from the existing Slack link</summary>

The existing Slack link is https://hackforla.slack.com/messages/C02509WHFQQ is redirected as shown:

![image](https://github.com/hackforla/website/assets/5314153/37210e51-b80e-4427-bc49-1ae3dd89f50f)

The new Slack link is https://hackforla.slack.com/archives/civictechjobs
</details>

**Comment by jdingeman** [comment]
@JessicaLucindaCheng - so for this one, am _**I**_ making the template? Or are we having another issue for the dev to create a template (as a level 2) and then having other devs actually make the issue based off of the template (level 1)?

**Comment by JessicaLucindaCheng** [comment]
@JessicaLucindaCheng 

> @JessicaLucindaCheng - so for this one, am _**I**_ making the template? Or are we having another issue for the dev to create a template (as a level 2) and then having other devs actually make the issue based off of the template (level 1)?

I'm going with having another issue for the dev to create a template (as a level 2) and then having other devs actually make the issue based off of the template (level 1).

**Comment by JessicaLucindaCheng** [comment]
@jdingeman 

> @JessicaLucindaCheng - so for this one, am _**I**_ making the template? Or are we having another issue for the dev to create a template (as a level 2) and then having other devs actually make the issue based off of the template (level 1)?

Having another issue for the dev to create a template (as a level 2) and then having other devs actually make the issue based off of the template (level 1).

**Comment by ExperimentsInHonesty** [comment]
````
### Prerequisite
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started page](https://www.hackforla.org/getting-started) and attend an onboarding session.
2. Before you claim or start working on an issue, please make sure you have read our [How to Contribute to Hack for LA Guide](https://github.com/hackforla/website/blob/7f0c132c96f71230b8935759e1f8711ccb340c0f/CONTRIBUTING.md).

### Overview
We need to ensure that links on the website resolve properly and are up to date.

### Action Items
- [ ] Change the GitHub link for [Replace PROJECT NAME] so it resolves to the correct url
    - **Line 54** under `_projects/[Replace PROJECT.MD file name]`

From:
```
url: "[Replace existing slack link]"
```
To:
```
url: "[Replace with new slack link]"
```
- [ ] Ensure that clicking the Slack link from the [projects page](https://www.hackforla.org/projects/) and [Replace with PROJECT NAME](https://www.hackforla.org/projects/[Replace PROJECT PAGE URL]) both resolve to the new address

### Resources/Instructions
- Directory to find the page in once you have it in your IDE: `_projects/[Replace PROJECT.MD file name]`
- [Projects page](https://www.hackforla.org/projects/)
- [Food Oasis page](https://www.hackforla.org/projects/[Replace PROJECT PAGE URL])
- Click on the filename below to see the source file code:
[Replace with link in the GitHub repo]
````

**Comment by roslynwythe** [comment]
@ExperimentsInHonesty @JessicaLucindaCheng This issue includes instructions as well as a template can we change the label to issue making level 1 and size 2 pt and then prioritize it?  Or should I close this one and create a new issue making level 1 issue?

**Comment by JessicaLucindaCheng** [comment]
@roslynwythe 
- > ExperimentsInHonesty @JessicaLucindaCheng This issue includes instructions as well as a template can we change the label to issue making level 1 and size 2 pt and then prioritize it? Or should I close this one and create a new issue making level 1 issue?

  - I went ahead and made quite a few edits to the Action Items and Template because it was confusing with references to GitHub links (I think this was from Justin copying a previous issue and pasting it here). In addition, I created issue #5716 as an example. Also, I went ahead and changed the labels from issue making level 2 to level 1 and size to 2pt. 

- I edited the action items to also add a `ready for dev lead` label on issues that are to be written and added this action item:
  > - [ ] In a comment to @hackforla/website-merge, ask a Merge Team member to review the issues you created to see if they are `ready for prioritization` and move this issue to the Questions / In Review column

  - Is this the process you want for reviewing issues?


- Also, I recommend the following to be done
  - [x] Confer with @ExperimentsInHonesty to
    - [x] Get approval on the template
    - [x] In addition, confirm the `Add the milestone 03.02 Onboarding flow` is the correct milestone to be added to the to-be-written issues

**Comment by roslynwythe** [comment]
@ExperimentsInHonesty @JessicaLucindaCheng this looks good to me.   I did make a minor change in the template for the good first issue, simplifying the Action item from 
```
- [ ] Change the Slack link for [Replace with PROJECT NAME] so it resolves to the updated URL. In `_projects/[Replace PROJECT.MD file name]`, for the Slack `url` under the `links` section, change 
```
to
```
In `_projects/[Replace PROJECT.MD file name]`, for the Slack `url` under the `links` section, change 
```

**Comment by JessicaLucindaCheng** [comment]
@ExperimentsInHonesty @roslynwythe I added this last Action Item to the template:
```
- [ ] Once your pull request is merged, go to #4564 and
  - [ ] Check off this issue under the _Dependency_ section
  - [ ] If all the issues in the _Dependency_ section are checked off, move #4564 to **Questions / In Review** column and add a **ready for dev lead** label to it
```

**Comment by roslynwythe** [comment]
@ExperimentsInHonesty I updated the issue so that the new Slack link is of the form https://hackforla.slack.com/archives/civictechjobs, but for the sake of consistency, should this epic also create issues to update the Slack link for the other projects that were already using "archives" in their link (for example in 311-data.md: 'https://hackforla.slack.com/archives/CJY4R382X')?

**Comment by ExperimentsInHonesty** [comment]
Sorry about the miscommunication.  We actually should stick with the symbolic link (the numbers) instead of the name, because the numbers keep working even when the name is changed.

**Comment by roslynwythe** [comment]
No problem, the issue is ready and creates links of the form https://hackforla.slack.com/archives/C02509WHFQQ

**Comment by marioantonini** [comment]
i. Availability: S, M, T 2-5 pm
ii. ETA: Mar 25th EOD
Note: Overall availability will be sketchy second half of next week (Mar 27 - Mar 30)

**Comment by marioantonini** [comment]
Greetings @hackforla/website-merge! As per one of the action items in this issue, could any of you review the issues created to see if they are ready for prioritization?
- [ ] #6511
- [ ] #6512
- [ ] #6513
- [ ] #6514
- [ ] #6515

Thanks!

**Comment by ExperimentsInHonesty** [comment]
I reviewed all the issues, and they look perfect.  I have prioritized them, I am moving this issue to the icebox.
